doc/changes/changes_0.1.0.md

```diff
@@ -1,8 +1,10 @@
-# Sphinx GitHub Pages Generator 0.1.0, released X-X-X
+# Sphinx GitHub Pages Generator 0.1.0, released 2022-06-09
 Code name: Initial implementation
 
 ## Summary
 
+This release provides an initial version of the Sphinx GitHub Pages Generator. It can be used to automatically build and commit or push the Sphinx-documentation of a project to a target GitHub branch. It Also automatically creates an index.html file listing the created releases that can be used as an entrypoint for GithubPages.
```

Comment:
```suggestion
This release provides an initial version of the Sphinx GitHub Pages Generator. It can be used to automatically build and commit or push the Sphinx-documentation of a project to a target GitHub branch. It also automatically creates an index.html file listing the created releases that can be used as an entrypoint for GithubPages.
```